librz/core/vmenus.c

```diff
@@ -2311,6 +2311,28 @@ static void config_visual_hit(RzCore *core, const char *name, int editor) {
 	}
 }
 
+static void show_config_options(RzCore *core, const char *opt) {
+	RzConfigNode *node = rz_config_node_get (core->config, opt);
+	if (node && !rz_list_empty (node->options)) {
+		int h, w = rz_cons_get_size (&h);
+		const char *item;
+		RzListIter *iter;
+		RzStrBuf *sb = rz_strbuf_new (" Options: ");
+		rz_list_foreach (node->options, iter, item) {
+			rz_strbuf_appendf (sb, "%s%s", iter->p? ", ": "", item);
+			if (rz_strbuf_length (sb) + 5 >= w) {
+				char *s = rz_strbuf_drain (sb);
```

Comment:
Mmm why not just appending `\n` to the existing `sb` instead of printing the existing one + creating a new one?

Reply:
I guess to be able to use `rz_strbuf_length ()`. To be honest, I think this output is not very good, and could be changed in the future (see my https://github.com/rizinorg/rizin/pull/351#issuecomment-759220657 comment). But it could be used as a base for the improvement.